Documentation/MigrateToComposer/MigrationSteps.rst

```diff
@@ -264,22 +297,32 @@ To complete our example ``composer.json``, it would look like this:
         "extra": {
             "typo3/cms": {
                 "cms-package-dir": "{$vendor-dir}/typo3/cms",
-                "web-dir": "web"
+                "web-dir": "public"
             }
         },
         "autoload": {
             "psr-4": {
-                "VendorName\\MySitepackage\\": "web/typo3conf/ext/my_sitepackage/Classes/",
-                "VendorName\\AnyOtherExtension\\": "web/typo3conf/ext/any_other_extension/Classes/"
+                "VendorName\\MySitepackage\\": "public/typo3conf/ext/my_sitepackage/Classes/",
+                "VendorName\\AnyOtherExtension\\": "public/typo3conf/ext/any_other_extension/Classes/"
             },
             "classmap": [
-                "web/typo3conf/ext/my_old_extension/pi1/",
-                "web/typo3conf/ext/my_old_extension/pi2/class.tx_myoldextension_pi2.php"
+                "public/typo3conf/ext/my_old_extension/pi1/",
+                "public/typo3conf/ext/my_old_extension/pi2/class.tx_myoldextension_pi2.php"
             ]
         }
     }
 
+.. note::
+
+    If you want to keep your :file:`typo3conf/ext` directory empty and ``autoload`` information only
+    in extensions' :file:`composer.json`, but not in your project's :file:`composer.json`,
+    there is an alternative way to include your individual extensions in the chapter
+    `completely clear "typo3conf/ext" folder <./BestPractices.html#completely-clear-typo3conf-ext-folder>`__
```

Comment:
This references work? Didn't know about them.

Reply:
It works in my local test rendering. Which way would you recommend to link between the documents?

![bildschirmfoto 2018-08-20 um 13 49 12](https://user-images.githubusercontent.com/16088567/44338947-ee437400-a47f-11e8-90de-4ffc8f7e2cd9.png)

Reply:
Thanks for that hint. I changed the links because the way you do it, looks more future proof. In my way I had concerns that I would forget to change the links, when I had to change any headline.

Reply:
This also has the huge benefit, that they are available via "intersphinx" from within foreign sphinx renderings. So another manual can link to the target and everytime its rendered, the link will be adjusted.